tests/test_wave_07.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+from swap_meet.vendor import Vendor
+from swap_meet.item import Item
+from swap_meet.clothing import Clothing
+from swap_meet.decor import Decor
+from swap_meet.electronics import Electronics
+
+def test_swap_by_age():
```

Comment:
Nice that you got started testing here.  You should also include edge-case tests with empty lists of inventory and where the ages are tied.